internal/pkg/print/print.go

```diff
@@ -85,3 +91,27 @@ func (p *Printer) Error(msg string) {
 func (p *Printer) OutOrStdout() io.Writer {
 	return p.Cmd.OutOrStdout()
 }
+
+// Prompts the user for confirmation.
+//
+// Returns nil only if the user (explicitly) answers positive.
+// Returns ErrAborted if the user answers negative.
+func (p *Printer) PromptForConfirmation(cmd *cobra.Command, prompt string) error {
```

Comment:
The printer already has a field for the command (`p.Cmd`), so we could probably remove it from the function arguments